catalog/ui/catalog-ui-search/src/main/webapp/component/list-item/list-item.hbs

```diff
@@ -44,6 +44,9 @@
             <div class="list-edit is-button" title="Edits the list." data-help="Edits the list.">
                 <span class="fa fa-pencil"></span>
             </div>
+            <div class="list-add is-button" title="Add items to the list." data-help="Import or manually add new items to the list.">
+                <span class="fa fa-plus"></span>
```

Comment:
Might be interesting to try the upload icon here.

Reply:
changed